index.js

```diff
@@ -17,7 +17,57 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+    Recipe.create()
+    const newRecipe = {
+      title: "Bolognese Pasta",
+      level: "Easy Peasy",
+      ingredients: ["pasta", "tomato sauce", "cheese"],
+      cuisine: "Italian",
+      dishType: "main_course",
+      image: "https://example.com/image.jpg",
+      duration: 30,
+      creator: "John Doe",
+    };
+    // return Recipe.create(newRecipe);
+
+  })
+  .then((createdRecipe) => {
+    console.log(`Created recipe: ${createdRecipe.title}`);
+  })
+  .then(() => {
+
+    return Recipe.insertMany(data);
+
+  })
+  .then(() => {
+    console.log(`titles, ${data[0].title}`);
+  })
+  .then(updatedRecipe => {
+    return Recipe.findOneAndUpdate({ title: "Rigatoni alla Genovese" }, { duration: 100 })
+
+  })
+  .then(() => {
+    console.log('Updated succesfull')
```

Comment:
You don't need a `.then()` for just a `console.log()`
```suggestion
    console.log('Updated succesfull')
    return Recipe.deleteOne({ title: "Carrot Cake" })
```